staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/authentication.go

```diff
@@ -31,17 +31,30 @@ import (
 // rest.Config generated by the resolver.
 type AuthenticationInfoResolverWrapper func(AuthenticationInfoResolver) AuthenticationInfoResolver
 
-// AuthenticationInfoResolver builds rest.Config base on the server name.
+// AuthenticationInfoResolver builds rest.Config base on the host or service
+// name.
 type AuthenticationInfoResolver interface {
-	ClientConfigFor(server string) (*rest.Config, error)
+	// ClientConfigFor builds rest.Config based on the host.
+	ClientConfigFor(host string) (*rest.Config, error)
+	// TODO(yguo0905): Remove ClientConfigForService once the SSH tunnels that
```

Comment:
I'd remove the TODO... the webhook registration API has URL and service reference... I'd anticipate both being represented here, even once the kube-apiserver tunneling dialer is not needed